### PR TITLE
Intial add of Comments and Questions Modules

### PR DESCRIPTION
**Pull request by vvnnzar**

### CODE REVIEW

File: models/Questions.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  {
+    sequelize,
+    timestamps: false,
+    freezeTableName: true,
+    underscored: true,
+    modelName: "category",
+  }
+);
+
+module.exports = Comments;
```
Comment (katemccoll) [inline models/Questions.js]: Questions :)

File: models/Questions.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        key: "id",
+      },
+    },
+  },
+  {
+    sequelize,
+    timestamps: false,
+    freezeTableName: true,
+    underscored: true,
+    modelName: "category",
```
Comment (katemccoll) [inline models/Questions.js]: Questions here too

File: models/Comments.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        key: "id",
+      },
+    },
+  },
+  {
+    sequelize,
+    timestamps: false,
+    freezeTableName: true,
+    underscored: true,
+    modelName: "category",
```
Comment (katemccoll) [inline models/Comments.js]: comment

**Review by katemccoll — COMMENTED** [review]
just those changes to be made :)

**Review by katemccoll — APPROVED** [review]
Looks great!

**Review by lewy192 — APPROVED** [review]
Great implementaion of sequelize.

Clean code.

:)

**Pull request merged**